fetch.bs

```diff
@@ -6631,6 +6705,64 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
 </ol>
 </div>
 
+<h3 id=deferred-fetching>Deferred fetching</h3>
+
+<p>Deferred fetches allow callers to request that a fetch is invoked at the latest possible moment,
+i.e. when a <a>fetch group</a> is <a for="fetch group">terminated</a>, or after a timeout after it
+is <a for="fetch group">deactivated</a>.
+
+<div algorithm="request-a-deferred-fetch">
+<p>To <dfn>request a deferred fetch</dfn> given a
+<a for=/>request</a> <var>request</var> and a null-or-{{DOMHighResTimeStamp}}
+<var>inactivityDeferredDelay</var> (default null):
+
+<ol>
+ <li><p>Assert: <var>request</var>'s <a for=request>client</a> is an
+ <a>environment settings object</a>.
+
+ <li>Let <var>totalScheduledDeferredBytesForOrigin</var> be 0.
+
+ <li>
+  <p>If <var>request</var>'s <a for=request>body</a> is not null then:
+
+  <ol>
+   <li><p>If <var>request</var>'s
+   <a for=request>body</a>'s <a for=body>length</a> is null, then throw a {{TypeError}}.
+
+   <li><p>Set <var>totalScheduledDeferredBytesForOrigin</var> to <var>request</var>'s
+   <a for=request>body</a>'s <a for=body>length</a>.
+  </ol>
+ </li>
+
+ <li><p><a for=list>For each</a> <a>deferred fetch record</a> <var>deferredRecord</var> in
+ <var>request</var>'s <a for=request>client</a>'s <a for=fetch>fetch group</a>'s
+ <a for="fetch group">deferred fetch records</a>: if <var>deferredRecord</var>'s
+ <a for="deferred fetch record">request</a>'s <a for=request>body</a> is not null and
+ <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>'s <a for=request>URL</a>'s
+ <a for=url>origin</a> is <a>same origin</a> with <var>request</var>'s <a for=request>URL</a>'s
+ <a for=url>origin</a>, then increment <var>totalScheduledDeferredBytesForOrigin</var> by
+ <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>'s <a for=request>body</a>'s
+ <a for=body>length</a>.
```

Comment:
```
then increment totalScheduledDeferredBytesForOrigin by deferredRecord’s  request’s  body’s  length
```

How to deal with Blob request, which length is null without fully reading it? Do we just allow them all by default?

Reply:
I think we should throw a network error when calling `fetchLater` with a non http(s) URL.

Reply:
```
I think we should throw a network error when calling fetchLater with a non http(s) URL.
```

Isn't it already specified to throw `TypeError`?

```
5. If request’s  URL’s  scheme is not an HTTP(S) scheme then throw a TypeError.
```
https://whatpr.org/fetch/1647/53e4c3d...71fd383.html#fetch-later-method


--

My comment [above](https://github.com/whatwg/fetch/pull/1647#discussion_r1311095896) is about the type of the request body of a `Post` fetchLater request being something that its length cannot be decided before sending. But I read somewhere that fetchLater request like `ReadableStream` should just be rejected. Could it be explicitly specifing here?

Reply:
Gotcha. Added another throw